general/development/policies/codingstyle/index.md

```diff
@@ -1778,7 +1778,7 @@ If you have a big task that is nearly done, apart a few TODOs, and you really wa
 
 There is a nice "to-do checker" reporting tool, restricted to admins and available via web @ [`lib/tests/other/todochecker.php`](https://github.com/moodle/moodle/blob/master/lib/tests/other/todochecker.php).
 
-Finally, don't forget to add any MDL-l2345 used by your TODOs (and @todos too, unless part of the [deprecation process](https://docs.moodle.org/dev/Deprecation), those are handled apart) to the "Review TODOs Epic": MDL-47779 (requires login to see the issues)
+Finally, don't forget to add any MDL-l2345 used by your TODOs (and @todos too, unless part of the [deprecation process](/general/development/policies/deprecation), those are handled apart) to the "Review TODOs Epic": MDL-47779 (requires login to see the issues)
```

Comment:
Just noticed that this is `MDL-L2345`. Not related to this patch but wanted to note so it doens't get lost.

Reply:
Well spotted! I've replaced "l" to "1" in this patch too ;-)